src/toolkit/matl_buy_policy.h

```diff
@@ -173,13 +193,22 @@ class MatlBuyPolicy : public Trader {
   void set_req_when_under(double x);
   void set_quantize(double x);
   void set_throughput(double x);
-  void set_active(int x);
-  void set_dormant(int x);
+  void set_default_distributions();
+  void init_active_dormant();
 
   ResBuf<Material>* buf_;
   std::string name_;
   double fill_to_, req_when_under_, quantize_, throughput_;
-  int active_, dormant_;
+  
+  double cumulative_capacity_;
```

Comment:
This isn't used.

Reply:
Oops, left over from thinking about adding cumulative capacity (which is the next step)